.github/workflows/build-main.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       - uses: ./.github/actions/create-release-apk
         with:
-          keystore: ${{ secrets.KEYSTORE_RELEASE }}
```

Comment:
temporary pipeline artifacts should only be signed with the debug keystore imo

Reply:
Could not get it to run on eclipse with the debug keystore

using: `secrets.KEYSTORE_DEBUG` lead to error `encodedString value is not set`

so signing temporarily with the release keystore